### ISSUE TITLE
Optimization : No such host & Non public IP

### ISSUE THREAD

**Issue by baylanger** [erkexzcx/stoppropaganda#95]
On `no such host` can we pause like we normally do for `Non public IP detected, DOS paused ...` ?

Below it shows the application tries non-stop with no pause to connect.

```
        "http://imctax.parus-s.ru": {
            "requests": 1531,
            "errors": 1531,
            "last_error_msg": "lookup imctax.parus-s.ru: no such host",
            "status": "Running",
```

Is this a bug below?  ca.ntssoft.ru & elkursk.ru reports "Non public IP detected" but only elkursk.ru shows "DOS is paused".  ca.ntssoft.ru is going non-stop.

```
        "http://ca.ntssoft.ru": {
            "requests": 1531,
            "errors": 1531,
            "last_error_msg": "CustomTCPDialer: Non public IP detected",
```

versus

```
        "http://elkursk.ru": {
            "requests": 0,
            "errors": 0,
            "last_error_msg": "",
            "status": "Non public IP detected, DOS paused for 14.5646685s",
```

**Comment by deputinizer** [comment]
Probably my fault. Well errors come from 2 sources:
- `website.ValidateDNS`
- `httpClient.DoTimeout`
Let's rename the errors to better track where they come from.

**Comment by deputinizer** [comment]
> Is this a bug below? ca.ntssoft.ru & elkursk.ru reports "Non public IP detected" but only elkursk.ru shows "DOS is paused".

Let's see, here's the fix for error messages:
https://github.com/erkexzcx/stoppropaganda/pull/93 https://github.com/erkexzcx/stoppropaganda/pull/93/commits/9f1d4b4fde3fe3e6aa7079e04e64ba069ce09104

**Comment by erkexzcx** [comment]
Should be fixed in v0.0.34

**Comment by baylanger** [comment]
Confirmed - this is fixed.
